src/hooks/useResponsiveLayout.ts

```diff
@@ -12,7 +12,7 @@ type ResponsiveLayoutResult = {
  */
 export default function useResponsiveLayout(): ResponsiveLayoutResult {
     const {isSmallScreenWidth} = useWindowDimensions();
-    const state = navigationRef?.getRootState();
+    const state = navigationRef?.current?.getRootState?.();
```

Comment:
```suggestion
    const state = navigationRef?.current?.getRootState();
```

I believe this is a typo.